R/get_sites.R

```diff
@@ -11,12 +11,13 @@
 #'   \code{\link[sbtools]{session_check_reauth}}
 #' @return a character vector of "site_root" titles (keys)
 #' @import sbtools
+#' @keywords internal
 #' @examples
 #' \dontrun{
 #' get_sites()
 #' get_sites(limit = 10)
 #' # get those sites that have water temperature
-#' get_sites(with_var_src = 'ts_wtr')
+#' get_sites(with_var_src = 'ts_wtr_nwis')
```

Comment:
i wonder if we should change `get_sites` to accept a true `var_src` and turn it into a `ts_name` internally. or change `with_var_src` to `with_ts_name`. either could be appropriate, seeing that `get_sites` is an internal function, though i think the former will be easier to use within list_sites.